packages/sandbox/ui/src/components/component-controls/ValueCheckbox.tsx

```diff
@@ -1,19 +1,32 @@
 import React from 'react'
-import { startWithType, mapHandlers, pureComponent, mapState, mapDebouncedHandlerTimeout } from 'refun'
+import { startWithType, mapHandlers, pureComponent, mapState, mapDebouncedHandlerTimeout, mapWithProps } from 'refun'
 import { Switch } from '../switch'
 import { SYMBOL_SWITCH } from '../../symbols'
 
 export type TValueCheckboxProps = {
   propPath: readonly string[],
   checkedPropValue: any,
   propValue: any,
+  propPossibleValues?: readonly any[],
   onChange: (propPath: readonly string[], propValue: any) => void,
 }
 
 const isDefined = (val: any): boolean => val !== false && val !== undefined
 
+const hasFalseInPosiibleValues = (propPossibleValues: readonly any[]): boolean => propPossibleValues.length === 2 && propPossibleValues.includes(true) && propPossibleValues.includes(false)
+
 export const ValueCheckbox = pureComponent(
   startWithType<TValueCheckboxProps>(),
+  mapWithProps(({ propPossibleValues, propValue }) => {
+    const valueCanBeFalse = propPossibleValues && hasFalseInPosiibleValues(propPossibleValues)
+
+    return (
+      {
+        valueCanBeFalse,
+        propValue: valueCanBeFalse && propValue === undefined ? true : propValue,
```

Comment:
this is in order to have it at first render `propValue` as `true` (isChecked) by default

Reply:
How does it work in case prop is not `required`? Do you know if it automatically shows up as `isChecked={true}` or does it have to be marked as `required` as well?
I guess I just don't know in Sandbox if the props that are not required are still passed as `undefined` or just not passed at all? Do you maybe know?